src/chapter10/01-setup.md

```diff
@@ -80,7 +80,7 @@ Create a new Supabase project, copy the database URL and create the following `.
 DATABASE_URL=$YOUR_DATABASE_URL_HERE
 ```
 
-> Of course, you need to specify the actual database URL you copied from Supabase instead of `$YOUR_DATABASE_URL_HERE`.
+> Of course, you need to specify the actual database URL you copied from Supabase instead of `$YOUR_DATABASE_URL_HERE`. Don't forget to replace any special charecters in your password from the final URL: https://developer.mozilla.org/en-US/docs/Glossary/Percent-encoding.
```

Comment:
Write one sentence per line (for git diff reasons).

Additionally, we prefer to explain concepts in the book, instead of just linking them, e.g.:

If your password has special characters like `:` or `/`, you will need to replace them with their respective percent-encodings (also called URL encodings).
For example, `:` should be replaced with `%3A` and `/` should be replaced with `%2F`.
You can read more about percent-encodings at https://developer.mozilla.org/en-US/docs/Glossary/Percent-encoding.